openstack/cronus/values.yaml

```diff
@@ -187,3 +188,30 @@ ingress-nginx:
     - name: tls-secret-cronus
       secret:
         secretName: tls-secret-cronus
+
+simulator:
+  enabled: true
+  image:
+    name: cronus-exporter-repository
+    pullPolicy: IfNotPresent
+    tag: latest
+    imagePullSecrets: []
+  statefulSet:
+    enabled: true
```

Comment:
I think there is no need to use statefulsets here. You don't use locks or anything else.

Reply:
I've disabled in secrets